web/libs/editor/src/components/SidePanels/DetailsPanel/RegionEditor.tsx

```diff
@@ -59,18 +52,21 @@ const IconMapping = {
 };
 
 const RegionEditorComponent: FC<RegionEditorProps> = ({ region }) => {
-  const fields: any[] = region.editableFields ?? [];
-  const isAudioModel = getType(region).name === "AudioRegionModel";
+  const isAudioRegion = isFF(FF_DEV_2715) && region.type === "audioregion";
+  const isTimelineRegion = region.type === "timelineregion";
+  const Component = isTimelineRegion ? TimelineRegionEditor : isAudioRegion ? AudioRegionProperties : RegionProperties;
 
-  const changeStartTimeHandler = (value: number) => {
-    region.setProperty("start", value);
-  };
+  return (
+    <Block name="region-editor" mod={{ disabled: region.isReadOnly() }}>
+      <Component region={region} />
+    </Block>
+  );
+};
 
-  const changeEndTimeHandler = (value: number) => {
-    region.setProperty("end", value);
-  };
+const RegionProperties = ({ region }: RegionEditorProps) => {
+  const fields: any[] = region.editableFields ?? [];
```

Comment:
```suggestion
  const fields = region.editableFields ?? [];
```
Does this not already pick up the updated type coming from your newly typed `MSTRegion`?

Reply:
Thank you! It's actually better without any with type inference!